workflow/rules/common.smk

```diff
@@ -459,6 +461,76 @@ def get_adapters(wildcards):
     return config["adapters"]["illumina-revelo"]
 
 
+def get_final_assemblies(wildcards):
+    if wildcards.assembly_type == "masked-assembly":
+        pattern = "results/{{date}}/contigs-masked/{sample}.fasta"
+    elif wildcards.assembly_type == "pseudo-assembly":
+        pattern = "results/{{date}}/contigs-masked/{sample}.fasta"
+
+    return expand(pattern, sample=get_samples_for_date(wildcards.date))
+
+
+def get_final_assemblies_identity(wildcards):
+    if wildcards.assembly_type == "masked-assembly":
+        pattern = "results/{{date}}/quast/masked/{sample}/report.tsv"
+    elif wildcards.assembly_type == "pseudo-assembly":
+        pattern = "results/{{date}}/quast/pseudoassembly/{sample}/report.tsv"
+
+    return expand(pattern, sample=get_samples_for_date(wildcards.date))
+
+
+def get_assemblies_for_submission(wildcards, agg_typ):
+    with checkpoints.rki_filter.get(
+        date=wildcards.date, assembly_type="masked-assembly"
+    ).output[0].open() as f:
+        masked_samples = pd.read_csv(f, squeeze=True).astype(str).to_list()
+
+    with checkpoints.rki_filter.get(
+        date=wildcards.date, assembly_type="pseudo-assembly"
+    ).output[0].open() as f:
+        pseudo_samples = pd.read_csv(f, squeeze=True).astype(str).to_list()
+
+    pseudo_assembly_pattern = "results/{{date}}/pseudoassembled-contigs/{sample}.fasta"
+    normal_assembly_pattern = "results/{{date}}/contigs-masked/{sample}.fasta"
```

Comment:
When I tried to solve the problem I thought of renaming all the different contigs from e.g. /peudoassambled-contigs/ /contigs/pseudo/, which would make the naming a little more consistent

Reply:
Good idea! Improved in 874adcf.